packages/core/src/components/LegacySearch/LegacySearch.tsx

```diff
@@ -113,6 +113,7 @@ const LegacySearch: VibeComponent<SearchProps, unknown> & {
         role="search"
         loading={loading}
         underline={type === SearchType.UNDERLINE}
+        iconsNames={iconNames}
```

Comment:
remove the eslint for unused var at [line 78](https://github.com/mondaycom/vibe/pull/2221/files#diff-f06badecccb1655de11b529c2bad5b3f9d88fe48331bb3f3857dbe8588583818R79)
also, should remove the `layout` from the Search fields now, right?

Reply:
or is it cool that it is the way it is right now and it is ok to leave it?

Reply:
It would be a breaking change to remove the `layout` now, no?

Reply:
meant in `SearchDefaultIconNames` but on 2nd thought it is gonna be removed soon with the entire Search component, so no rush on that one

Reply:
meant in `SearchDefaultIconNames` but on 2nd thought it is gonna be removed soon with the entire `LegacySearch` component, so no rush on that one